TruckMuncher/VendorMapViewController.swift

```diff
@@ -19,21 +19,25 @@ class VendorMapViewController: UIViewController, MKMapViewDelegate, CLLocationMa
     @IBOutlet var servingModeSwitch: UISwitch!
     var pulsingLayer : ServingModePulse!
     let deltaDegrees = 0.005
+    let selectionViewHeight = CGFloat(45.0)
     var locationManager: CLLocationManager!
-    var truck: RTruck
```

Comment:
It makes zero functional difference, but you could also have made this a computed property.
```
var truck: RTruck {
    get {
        return trucks[selectedTruckIndex]
    }
}
```
Dont worry about it though